dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/message/EmailMessageSender.java

```diff
@@ -306,12 +308,17 @@ private HtmlEmail getHtmlEmail( String hostName, int port, String username, Stri
 
     private String renderPlainContent( String text, User sender )
     {
-        return sender == null ? text
+        String content = sender == null ? text
             : (text + LB + LB + sender.getName() + LB
-                + (sender.getOrganisationUnitsName() != null ? (sender.getOrganisationUnitsName() + LB)
-                    : StringUtils.EMPTY)
-                + (sender.getEmail() != null ? (sender.getEmail() + LB) : StringUtils.EMPTY)
-                + (sender.getPhoneNumber() != null ? (sender.getPhoneNumber() + LB) : StringUtils.EMPTY));
+                + getNullSafe( sender.getOrganisationUnitsName() )
+                + getNullSafe( sender.getEmail() )
+                + getNullSafe( sender.getPhoneNumber() ));
+        return new String( content.getBytes( StandardCharsets.UTF_8 ), StandardCharsets.UTF_8 );
```

Comment:
I am not quite following the idea behind this recreation of a `String`. First it encodes the `content` in UTF-8 to then recreate a string instance from the UTF-8 bytes. That should give us the original `content` value. Is this some form of indirect validation?

Reply:
The idea is to convert a non-unicode text to Unicode. 
If the original text is already in UTF-8 then it should return same value.